src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write code here
+const inputs = document.querySelectorAll('input');
+
+for (const input of inputs) {
+  const label = document.createElement('label');
+
+  label.className = 'field-label';
+
+  const id = input.id;
+
+  label.htmlFor = id;
```

Comment:
You use id only once, so you didn't really need to create a variable for it
```suggestion
  label.htmlFor = input.id;